bigtable/bigtable-proxy/src/main/java/com/google/cloud/bigtable/examples/proxy/channelpool/DataChannel.java

```diff
@@ -148,6 +151,56 @@ private void warm() {
     }
   }
 
+  private ListenableFuture<PingAndWarmResponse> sendPingAndWarm(PingAndWarmRequest request) {
+    CallLabels callLabels =
+        CallLabels.create(
+            BigtableGrpc.getPingAndWarmMethod(),
+            Optional.of("bigtableproxy"),
+            Optional.of(request.getName()),
+            Optional.of(request.getAppProfileId()));
+    Tracer tracer = new Tracer(metrics, callLabels);
+
+    CallOptions callOptions =
+        CallOptions.DEFAULT
+            .withCallCredentials(callCredentials)
+            .withDeadline(Deadline.after(1, TimeUnit.MINUTES));
+    callOptions = tracer.injectIntoCallOptions(callOptions);
+
+    ClientCall<PingAndWarmRequest, PingAndWarmResponse> call =
+        inner.newCall(BigtableGrpc.getPingAndWarmMethod(), callOptions);
+
+    Metadata metadata = new Metadata();
+    metadata.put(
+        CallLabels.REQUEST_PARAMS,
```

Comment:
do we need user agent or api client?

Reply:
UserAgent is set during channel construction time. api_client is only for gapic like clients